### PR TITLE
adding more detail

### PR DESCRIPTION
**Pull request by rajrsingh**

### CODE REVIEW

**Review by daimor — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,13 +34,30 @@ For example:
...
 ```
 
-This extension helps users add server definitions to their [user or workspace settings](https://code.visualstudio.com/docs/getstarted/settings) by editing JSON files.
+## Storing passwords securely
+
+This extension adds the command `InterSystems Server Manager: Store Password in Keychain` to the Command Palette, which offers a quickpick of defined servers, then prompts for a password to store. This facility should be used instead of the plaintext `password` property of a server's definition, which has been deprecated.
+
+**Usage:**
+
+1. Bring up the command palette by typing CTRL-Shift-P (Mac: Command-Shift-P)
```
Comment (gjsjohnmurray) [inline README.md:43]: VS Code docs writes these as Ctrl+Shift+P / Cmd+Shift+P. It also capitalizes Command Palette. I suggest we do the same. There's a second occurrence later in your changes. Also, on Windows at least, F1 launches the Command Palette. Docs say the same happens on macOS. I find that a lot easier, and more memorable.

File: README.md [hunk 2]
```diff
@@ -26,13 +34,30 @@ For example:
...
+
+## Removing a stored password
+
+The command `InterSystems Server Manager: Clear Password from Keychain` removes a stored password.
 
-It adds the command `InterSystems Server Manager: Store Password in Keychain` to the Command Palette, which offers a quickpick of defined servers, then prompts for a password to store. This facility should be used instead of the plaintext `password` property of a server's definition, which has been deprecated.
+**Usage:**
 
-A command `InterSystems Server Manager: Clear Password from Keychain` removes a stored password.
+1. Bring up the command palette by typing CTRL-Shift-P (Mac: Command-Shift-P)
```
Comment (gjsjohnmurray) [inline README.md:56]: As above

File: README.md [hunk 3]
```diff
@@ -1,7 +1,15 @@
 # InterSystems® Server Manager
-This is a VS Code helper extension that contributes settings which define connections to [InterSystems](https://www.intersystems.com/) servers.
+
+This is a VS Code helper extension that helps configure connections to [InterSystems](https://www.intersystems.com/) servers in VS Code extensions such as the [ObjectScript Extension for VS Code](https://github.com/intersystems-community/vscode-objectscript).
+
+It also enables the secure storage of passwords using the underlying operating system's native key store.
```
Comment (gjsjohnmurray) [inline README.md:5]:
```suggestion
It also enables the secure storage of passwords using the underlying operating system's native keystore.
```

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
Thanks for adding the extra detail. When incorporating my feedback please also add periods at the end of each list item.

File: README.md (hunk 3 above)
Comment (rajrsingh) [reply README.md:5]: I wrote keystore in my DC post, and Jeff Fried changed it to 2 words, so I agree with you!

**Review by gjsjohnmurray — APPROVED** [review]
LGTM

**Pull request merged**